### DISCUSSION TITLE
Is having Dagster Cloud specific documentation on the main documentation site confusing?

### DISCUSSION THREAD

**Discussion by joshuataylor** [dagster-io/dagster#11034] [General]
When trying to find something on the documentation site, having to dig through documentation and then finding it's "Dagster Cloud only" (which is noted clearly at the top of the page) is confusing.

Top of page message:
<img width="707" alt="image" src="https://user-images.githubusercontent.com/225131/206843451-ea12669e-02b7-404b-baec-be7a0e5a4aa3.png">

Search *does* show Cloud:
<img width="479" alt="image" src="https://user-images.githubusercontent.com/225131/206843562-670ca1ec-bef6-4f02-9ffd-afdc6718c465.png">

Thoughts?

**Comment by erinkcochran87** [comment]
Hey @joshuataylor - could you elaborate on what you were looking for / trying to do, so I can better understand where the confusion is?
